.eslintrc

```diff
@@ -23,7 +23,7 @@
           "ignoreTemplateLiterals": true,
           "ignoreComments": true,
         }],
-        "operator-linebreak": [2, "after"],
+        "operator-linebreak": [2, "after", {"overrides": {"?": "before", ":": "before" }}],
```

Comment:
if this line is removed, isn’t this what the Airbnb config already mandates?

Reply:
Airbnb config uses "before".

https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/rules/style.js#L422

Reply:
oh right. how much would change if we switched to that? Alternately, could we use before here, and use explicit overrides for the ones we want set to after, to match the Airbnb config?

Reply:
There will be 840 lines of codes changed if that said option were to be switched to Airbnb style.
https://github.com/golopot/eslint-plugin-react/commit/267854438506153adb49a4ee72ab74306a3c0271
I would like to see the option being kept as is, due to my personal preference for "after" style, and for the sake of avoiding git diffs in history.

Reply:
I prefer the "before" style personally; but even if we wanted to continue setting operators like `&&` to "after", that seems better to me (ie, to be overriding airbnb's "before" setting for specific operators, instead of replacing them)

Reply:
I suppose the next action is you come up with an configuration and I will accept it. I believe you.